core/base/constant.go

```diff
@@ -29,4 +29,6 @@ const (
 	DefaultIntervalMsTotal uint32 = 10000
 
 	DefaultStatisticMaxRt = int64(60000)
+
+	DefaultStatisticMinConcurrency = int32(0)
```

Comment:
Maybe :`DefaultStatisticMaxConcurrency `

Reply:
I mainly refer to the default value naming of the minRt() method here, and here is mainly to calculate the maximum value, then the default value should be the minimum value. So the name is DefaultStatisticMinConcurrency.

Reply:
Maybe the constant `DefaultStatisticMinConcurrency` is not necessary, as the initial concurrency should be 0 apparently.

Reply:
done